skeleton-typescript-asp.net5/src/skeleton-navigation-typescript-vs/tsconfig.json

```diff
@@ -22,7 +22,8 @@
     "./wwwroot/jspm_packages/**/*.d.ts"
   ],
   "exclude": [
-    "node_modules"
+    "node_modules",
+    "typings/main"
```

Comment:
I think `typings/main.d.ts` should be excluded too. This line excludes only the folder.